ydb/docs/en/core/reference/ydb-sdk/health-check-api.md

```diff
@@ -349,8 +344,8 @@ The status (severity) of the current issue:
 
   - Primary Source: `/sys/fs/cgroup/cpu.max`
   - Fallback Sources: `/sys/fs/cgroup/cpu/cpu.cfs_quota_us`, `/sys/fs/cgroup/cpu/cpu.cfs_period_us`
--
- The number of cores is calculated by dividing the quota by the period (quota / period).
+
+The number of cores is calculated by dividing the quota by the period $(quota / period)$.
```

Comment:
```suggestion
The number of cores is calculated by dividing the quota by the period `(quota / period)`.
```

Reply:
@anton-bobkov that was intentional, see https://diplodoc.com/docs/ru/syntax/additional#formulas